README.rst

```diff
@@ -22,13 +25,13 @@ OpenWISP 2 monitoring module (Work in progress).
 Available Features
 ------------------
 
-* Displays ``Uptime``, ``Memory Status``, ``Load Status``, ``Addresses``, ``Neighbours`` of the **Device**
-* Displays ``Monitoring Graphs`` namely ``Uptime``, ``Packet Loss``, ``Round Trip Time``, ``Traffic`` in the form of lively charts
+* Displays `Uptime, Memory Status, Load Status, Addresses, Neighbours of the **Device** <docs/device_status.png>`_
```

Comment:
\*\*Device\*\*  -> Device (the link shows the asterisks, not bold)

Reply:
oops